lib/user.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/joke'
+
+class User
+attr_reader :name
+attr_accessor :jokes
```

Comment:
same here-- no reason for accessor, since we're not setting this value from outside the class